exoflex: Can't resolve '@ptomasroos/react-native-multi-slider'

This is only happening on the web, works fine on mobile. Maybe we should check whether `@ptomasroos/react-native-multi-slider` does have web support or not.

```shell
Failed to compile.
/Users/Me/projects/opensource/infra/packages/exoflex/src/components/Slider.tsx
Module not found: Can't resolve '@ptomasroos/react-native-multi-slider' in '/Users/Me/projects/opensource/infra/packages/exoflex/src/components'
```

## Preview
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/4923122/64243919-5ef5eb00-cf32-11e9-9980-412e60cf2761.png">

--- a/packages/exoflex/example/webpack.config.js
+++ b/packages/exoflex/example/webpack.config.js
@@ -22,6 +22,9 @@ module.exports = async function(env, argv) {
    * @see https://stackoverflow.com/questions/57455200/cant-use-hooks-with-my-react-component-library-invariant-violation-invalid-hoo
    */
   config.resolve.alias['react'] = path.resolve('./node_modules/react');
+  config.resolve.alias['@ptomasroos/react-native-multi-slider'] = path.resolve(
+    './node_modules/react-native-multi-slider',
+  );
 
   // Add rule to transform exoflex files before loading it.
   config.module.rules = [

